### ISSUE TITLE
Deploy the reference CHC service

### ISSUE THREAD

**Issue by ThetaSinner** [holochain/holochain-infra#155]
See https://github.com/holochain/hc-chc-service

We need to find a place, or make a place, to get this running. It needs to be publicly reachable and ideally run on infrastructure with consistent performance. I.e. not colocated if we can avoid it.

It's just a web service, and it stores its data in memory. So, it doesn't need too much compute. At least that's my expectation. It will be handling requests from a pool of HPs in parallel but I'm thinking it can manage well enough with a couple of cores and lots of threads.

**Comment by steveej** [comment]
@ThetaSinner just arrived at this in my notification emails. is this still relevant?

**Comment by ThetaSinner** [comment]
No longer relevant, the CHC reference service has been containerized and deployed on Digital Ocean
